crates/trie/common/src/proofs.rs

```diff
@@ -52,7 +48,9 @@ impl AccountProof {
     pub fn set_proof(&mut self, proof: Vec<Bytes>) {
         self.proof = proof;
     }
+}
 
+impl<A: Account + Default + Copy> AccountProof<A> {
```

Comment:
I consider this temp debt, which should go away if we make a few improvements to the account trait, like merging it with TrieAccount